module/batch-action/src/Infrastructure/Persistence/Repository/DbalBatchActionRepository.php

```diff
@@ -140,6 +140,20 @@ public function markEntry(BatchActionId $id, AggregateId $resourceId, array $mes
         );
     }
 
+    public function isProcessEnded(BatchActionId $id): bool
+    {
+        $query = $this->connection->createQueryBuilder();
+
+        $result = $query->select("bool_and(success)")
```

Comment:
it should check if not null, not the success itself  - false on success also means entry is processed